doc/source/user/c-info.python-as-glue.rst

```diff
@@ -407,13 +407,12 @@ conversion of the .pyf file to a .c file is handled by `numpy.disutils`.
 Conclusion
 ----------
 
-The interface definition file (.pyf) is how you can fine-tune the
-interface between Python and Fortran. There is decent documentation
-for f2py found in the numpy/f2py/docs directory where-ever NumPy is
-installed on your system (usually under site-packages). There is also
-more information on using f2py (including how to use it to wrap C
-codes) at https://scipy-cookbook.readthedocs.io under the "Interfacing
-With Other Languages" heading.
+The interface definition file (.pyf) is how you can fine-tune the interface
+between Python and Fortran. There is decent documentation for f2py found on the
+NumPy documentation site at https://numpy.org/devdocs/f2py/index.html. There is
+also more information on using f2py (including how to use it to wrap C codes) at
+https://scipy-cookbook.readthedocs.io under the "Interfacing With Other
+Languages" heading.
```

Comment:
I'd actually suggest including a label for the f2py main documentation page and referencing it here: 

```suggestion
The interface definition file (.pyf) is how you can fine-tune the interface
between Python and Fortran. There is decent documentation for f2py at
:ref:`f2py`. There is also more information on using f2py (including how to use
it to wrap C codes) at the `"Interfacing With Other Languages" heading of the
SciPy Cookbook.
<https://scipy-cookbook.readthedocs.io/items/idx_interfacing_with_other_languages.html>`_
```

and adding `.. _f2py:` to the top of `doc/source/f2py/index.rst`